doc/source/user_guide/fields/field_data.rst

```diff
@@ -95,6 +95,23 @@ To obtain face connectivity data, specify ``FacesConnectivity`` as the ``data_ty
   >>> faces_connectivity_data["inlet"].connectivity[5]
   array([ 4,  5, 12, 11])
 
+You can also query the face connectivity data in raw format:
+
+.. code-block:: python
+
+  >>> faces_connectivity_request = SurfaceFieldDataRequest(
+  >>>     surfaces=[VelocityInlet(settings_source=solver_session, name="inlet")],
+  >>>     data_types=[SurfaceDataType.FacesConnectivity]
+  >>>     raw_data=True,
+  >>> )
+  >>> faces_connectivity_data = field_data.get_field_data(faces_connectivity_request)
+
+  # FacesConnectivity provides indices of vertices for each face. For example:
+  # Face 6 is connected to vertices 4, 5, 12, and 11.
```

Comment:
@prmukherj Is the connection between the comment and te code/output obvious?

Reply:
Thank you @seanpearsonuk. This was  an error in the doc. Fixed it now.